extensions/blocks/contact-form/variations.js

```diff
@@ -33,6 +33,29 @@ const variations = [
 			],
 		],
 	},
+	{
```

Comment:
The new variation seems a bit off for me (WP 5.5, no Gutenberg):

![image](https://user-images.githubusercontent.com/426388/89921090-8b63be00-dbfd-11ea-8c81-31df017bd44a.png)

Reply:
This is fixed now. the viewport of the SVG was slightly off.

Reply:
![image](https://user-images.githubusercontent.com/47211808/89992496-9eba6c00-dc85-11ea-95e9-2c246d4b2841.png)